store/index.js

```diff
@@ -195,21 +194,43 @@ export default function configureStore(initialState) {
             },
             _stateGetter: (state, key) => {
                 if (key.indexOf('storage:') === 0) {
-                    state.storage = state.storage || {storage: {}};
-                    return state.storage.storage[key.substr(8)];
+                    return state.storage?.storage?.[key.substr(8)];
                 }
                 return state[key];
             },
             _stateSetter: (state, key, value) => {
+                // eslint-disable-next-line no-process-env
+                if (process.env.NODE_ENV === 'production') {
+                    if (key.indexOf('storage:') === 0) {
+                        state.storage = state.storage || {storage: {}};
+                        state.storage.storage[key.substr(8)] = value;
+                    } else {
+                        state[key] = value;
```

Comment:
Note that previously `state[key] = value` occurred /in addition/ to the redirected overwrite. I tested this and couldn't find any reason to do this, so removed the spurious write.